internal/operators/mtv/operator.go

```diff
@@ -0,0 +1,167 @@
+package mtv
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kelseyhightower/envconfig"
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/internal/featuresupport"
+	"github.com/openshift/assisted-service/internal/operators/api"
+	"github.com/openshift/assisted-service/internal/operators/cnv"
+	"github.com/openshift/assisted-service/models"
+	"github.com/openshift/assisted-service/pkg/conversions"
+	logutil "github.com/openshift/assisted-service/pkg/log"
+	"github.com/sirupsen/logrus"
+)
+
+type operator struct {
+	log    logrus.FieldLogger
+	config *Config
+}
+
+var Operator = models.MonitoredOperator{
+	Name:             Name,
+	Namespace:        Namespace,
+	OperatorType:     models.OperatorTypeOlm,
+	SubscriptionName: Subscription,
+	TimeoutSeconds:   60 * 60,
+}
+
+func NewMTVOperator(log logrus.FieldLogger) *operator {
+	cfg := Config{}
+	err := envconfig.Process(common.EnvConfigPrefix, &cfg)
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+	return &operator{
+		log:    log,
+		config: &cfg,
+	}
+}
+
+// GetName reports the name of an operator this Operator manages
+func (o *operator) GetName() string {
+	return Operator.Name
+}
+
+func (o *operator) GetFullName() string {
+	return FullName
+}
+
+func (o *operator) GetDependencies(cluster *common.Cluster) ([]string, error) {
+	return []string{cnv.Operator.Name}, nil
+}
+
+// GetClusterValidationID returns cluster validation ID for the Operator
+func (o *operator) GetClusterValidationID() string {
+	return string(models.ClusterValidationIDMtvRequirementsSatisfied)
+}
+
+// GetHostValidationID returns host validation ID for the Operator
+func (o *operator) GetHostValidationID() string {
+	return string(models.HostValidationIDMtvRequirementsSatisfied)
+}
+
+// ValidateCluster verifies whether this operator is valid for given cluster
+func (o *operator) ValidateCluster(_ context.Context, cluster *common.Cluster) (api.ValidationResult, error) {
+	if !featuresupport.IsFeatureCompatibleWithArchitecture(models.FeatureSupportLevelIDCNV, cluster.OpenshiftVersion, cluster.CPUArchitecture) {
```

Comment:
from which openshift verison it is supported? 
please add a check for min  openshift version